google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/TopicAdminSettings.java

```diff
@@ -91,7 +90,7 @@
   }
 
   /** Returns the object with the settings used for calls to publish. */
-  /* package-private */ BatchingCallSettings<PublishRequest, PublishResponse> publishSettings() {
+  /* package-private */ UnaryCallSettings<PublishRequest, PublishResponse> publishSettings() {
```

Comment:
Temporary hand-edits. If the rest of the change looks good, we can merge as-is, then update GAPIC config and regen.

Reply:
Could you also create the CL for updating the GAPIC config? Will stripping out batching break any other languages?